L1Trigger/L1TMuonEndCap/src/Forest.cc

```diff
@@ -21,6 +21,7 @@
 
 #include "TRandom3.h"
 #include "TStopwatch.h"
+#include "TROOT.h"
```

Comment:
This whole class will go away when the new CondFormat is used instead.... but will do.